spouts/rss/feed.php

```diff
@@ -243,7 +243,7 @@ public function getContent() {
      * @return string icon url
      */
     public function getIcon() {
-        if (isset($this->faviconUrl)) {
+        if ($this->faviconUrl !== '') {
```

Comment:
I have no argument for or against this. The test must match the initial definition, otherwise it doesn't work, that's all.